uaclient/cli.py

```diff
@@ -1581,6 +1638,27 @@ def action_status(args, *, cfg):
     return ret
 
 
+def action_system(args, *, cfg, **kwargs):
+    """Perform the system action.
+
+    :return: 0 on success, 1 otherwise
+    """
+    _print_help_for_subcommand(
+        cfg, cmd_name="system", subcmd_name=args.command
+    )
+    return 0
+
+
+def action_system_reboot_required(args, *, cfg: config.UAConfig):
+    result = call_api("u.pro.security.status.reboot_required.v1", [], cfg=cfg)
```

Comment:
let's just call the python function directly instead
```suggestion
    from uaclient.api.u.pro.security.status.reboot_required.v1 import reboot_required
    result = reboot_required()
```

Reply:
The only issue here is that we would be using the "private" function to avoid creating an extra unnecessary `UAConfig` instance, but this is a minor issue